requirements.txt

```diff
@@ -1,8 +1,9 @@
 -c constraints.txt
-ansible
-configparser
-flake8-filename
 flake8-pytest-mark
-magic-marker
 molecule
+moleculerize
 pytest-rpc
+pytest-zigzag
+magic-marker
+flake8-filename
+ansible
```

Comment:
Suggestion: keep your requirements and constraints file alphabetized so you don't accidentally specify the same package twice. We had this problem in ZigZag which caused some bizarre behavior behavior.

Reply:
👍 will update in the next push